cc_unitsearch.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import json
+
+from PyQt5.QtWidgets import QApplication, QWidget, QLineEdit, QTableView, QHeaderView, QVBoxLayout
+from PyQt5.QtCore import Qt, QSortFilterProxyModel, QRegExp
+from PyQt5.QtGui import QStandardItemModel, QStandardItem
+
+def pullFromJson():
+    with open('CC_UNITS.json') as json_file:
+        data = json.load(json_file)
+        
+        return data
+
+class CustomFilter(QSortFilterProxyModel):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.role = Qt.DisplayRole
+
+    def filterAcceptsRow(self, sourceRow, sourceParent):
```

Comment:
What is this function for?

Reply:
It's to overwrite Qt's default filter implementation

Reply:
Oh it's like an override function?

Reply:
Yup.

Reply:
Sad that Python has no way of marking overrides. Resolved then